### DISCUSSION TITLE
Lightning address payment

### DISCUSSION THREAD

**Discussion by FiliTol** [ACINQ/phoenixd#30] [Q&A] [answered]
Looking at the phoenix-cli help it seems that  it's not possible to pay some sats to a lnaddress. Is it possible to do it with the current phoenixd version? Will it be possible eventually?

**Comment by pm47** [comment, marked as answer]
It's not supported but we should be able to add it easily. Probably not the whole LNURL suite but at least LNURL-pay. Will look into it!

**Comment by tipogi** [comment]
If you want LNRUL, there is some work around to implement against phoenixd server (I have not try yet): [phonenixd-lnurl](https://github.com/AngusP/phoenixd-lnurl)

**Reply by FiliTol** [reply]
Thanks for the suggestion! It seems that the phoenixd-lnurl wrapper only allows for receiving sats with LNURL, since it does not implement arguments for the phoenix-cli.
